packages/embedder/resetStalledJobs.ts

```diff
@@ -0,0 +1,13 @@
+import ms from 'ms'
+import getKysely from 'parabol-server/postgres/getKysely'
+
+export const resetStalledJobs = () => {
+  setInterval(async () => {
+    const pg = getKysely()
+    await pg
```

Comment:
-1

Increment the retry counter here, also consider adding a state message

Reply:
done!